build_tools/cmake/iree_configure_testing.cmake

```diff
@@ -22,7 +22,7 @@ set(IREE_TEST_TMPDIR_ROOT "${IREE_BINARY_DIR}/test_tmpdir")
 #   TEST_NAME: the test name, e.g. iree/base/math_test
 function(iree_configure_test TEST_NAME)
   set(_TEST_TMPDIR "${IREE_TEST_TMPDIR_ROOT}/${TEST_NAME}_test_tmpdir")
-  set_property(GLOBAL APPEND PROPERTY IREE_TEST_TMPDIRS ${_TEST_TMPDIR})
+  # set_property(GLOBAL APPEND PROPERTY IREE_TEST_TMPDIRS ${_TEST_TMPDIR})
```

Comment:
intended?

Reply:
(that was so I could test the zero-dir case locally). Fixed.